geochatt/__init__.py

```diff
@@ -76,20 +76,87 @@ def get_address(longitude, latitude, max_distance=0.0001):
         return parcel_strtree["geoms"][nearest_geom]
 
 
+# Create dictionary of cardinal directions that may appear in addresses with their abbreviations
+cardinal_directions = {
+    "NORTH": "N", "NORTHEAST": "NE", "EAST": "E", "SOUTHEAST": "SE", 
+    "SOUTH": "S", "SOUTHWEST": "SW", "WEST": "W", "NORTHWEST": "NW"
+}
+# Create dictionary that contains all USPS street suffixes and their abbreviations
+street_suffixes = {
```

Comment:
Wow.  This is awesome!  I didn't know this list actually existed until now!

Just a small request, could you add a comment with a url to the data source?  I'm mostly asking because I'm curious and I'm sure other contributors may have the same question some day.